audits/bravo.py

```diff
@@ -340,19 +343,20 @@ def get_test_statistics(self, margins, sample_results):
         # Handle the no-losers case
         if not losers:
             for winner in winners:
-                T[(winner,)] = 1
+                T[(winner, )] = 1
 
         for cand, votes in sample_results.items():
             if cand in winners:
                 for loser in losers:
-                    T[(cand, loser)] *= (winners[cand]['swl'][loser]/0.5)**votes
+                    T[(cand,
+                       loser)] *= (winners[cand]['swl'][loser] / 0.5)**votes
```

Comment:
And again (I'm not gonna beat a dead horse, just wanted to provide some pointers to places where I'm not sure yapf is making the code more readable)